cpg-language-python/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonFrontendTest.kt

```diff
@@ -1376,6 +1377,93 @@ class PythonFrontendTest : BaseTest() {
         assertEquals(2L, fStmtRhsThird.value)
     }
 
+    @Test
+    fun testFormattedValues() {
+        val topLevel = Path.of("src", "test", "resources", "python")
+        val tu =
+            analyzeAndGetFirstTU(
+                listOf(topLevel.resolve("datatypes.py").toFile()),
+                topLevel,
+                true
+            ) {
+                it.registerLanguage<PythonLanguage>()
+            }
+        assertNotNull(tu)
+        val namespace = tu.namespaces.singleOrNull()
+        assertNotNull(namespace)
+
+        // Test for g = f'Number: {42:.2f}'
+        val gStmt = namespace.statements[6]
```

Comment:
I'm not too happy with the index access. However, I cannot provide a good alternative. The best thing I can come up with is:
```python
val gStmt = tu.refs["g"]?.astParent
```
Do you think this is better? Unfortunately, we lack any means to get the declaration here.

Reply:
Or better `tu.variables["g"]?.astParent`